tools/fetch-stats.js

```diff
@@ -57,8 +57,12 @@ async function fetchStatsFromGitHubAPI() {
         }
     });
 
-    const [currentRelease] = repository.releases.nodes;
-    const latestRelease = repository.releases.nodes.find(({ isPrerelease }) => !isPrerelease);
+    const releases = repository.releases.nodes.toSorted(
```

Comment:
Can you add a comment explaining why we're doing this?

Reply:
I've now added a comment explaining this.